examples/electra_style_qa/handler.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import sys
+
+import torch
+from transformers import (AutoConfig, AutoTokenizer, AutoModelForQuestionAnswering, QuestionAnsweringPipeline)
+
+from dynalab.handler.base_handler import BaseDynaHandler
+from dynalab.tasks.qa import TaskIO
+
+
+class Handler(BaseDynaHandler):
+
+    def initialize(self, context):
+        """
+        Load model and tokenizer files.
+        """
+        self.taskIO = TaskIO()
+        model_pt_path, _, device_str = self._handler_initialize(context)
+        config = AutoConfig.from_pretrained("config.json")
+        device = -1 if device_str == 'cpu' else int(device_str[-1])
+        self.pipeline = QuestionAnsweringPipeline(
+            model=AutoModelForQuestionAnswering.from_pretrained(model_pt_path, config=config),
+            tokenizer=AutoTokenizer.from_pretrained('.'), 
+            device=device
+        )
+        self.initialized = True
+
+    def preprocess(self, data):
+        """
+        Preprocess data into a format that the model can do inference on.
+        """
+        example = self._read_data(data)
+        return {
+            'context': example['context'],
+            'question': example['question']
+        }
+
+    def inference(self, input_data):
+        """
+        Run model using the processed data.
+        """
+        result = self.pipeline(input_data)
+        return result
+
+    def postprocess(self, inference_output, data):
+        """
+        Post process inference output into a response.
+        """
+        result = inference_output
+        answer = result['answer']
+        conf = result['score']
+
+        response = dict()
+        example = self._read_data(data)
+        response["id"] = example["uid"]
+        response["answer"] = answer if answer != '[CLS]' else ''
```

Comment:
Does the pipeline already take care of this CLS case?

Reply:
Nice spot, updated!